about.html

```diff
@@ -0,0 +1,311 @@
+                            and the community is growing. With the support of organizations and initiatives like those mentioned
+                            before, more and more African women are entering the technology industry and making significant
+                            contributions.
+
+
+
+
+                        </p>
+
+                    </div>
+                </div>
+                <div class="row border_line">
+                    <div class="col-4"></div>
+                    <div class="col-5 about_midlle_image">
+                        <img src="images/women.png" alt="logo bottom">
+                    </div>
+                    <div class="col-2"></div>
+                </div>
+            </div>
+            <!-- -----------Submit section ---------------------->
+            <div class="container-fluid about_submit_section">
+                <div class="row">
+                    <div class="col-4"></div>
+                    <div class="col-5 about_header_submit">
+                        <h2>View Past African Women in Tech Summits
+                        </h2>
+                        <div class="row">
+                            <div class="col-2"></div>
+                            <div class="col-10">
+                                <div class="submit_divider"></div>
+                            </div>
+                        </div>
+                    </div>
+
+
+                </div>
+                
+
+                <!--Image sections-->
+                <div class="container-fluid images-section" id="show_images">
+                    <div class="row img-mobile">
+                        <div class="col-1 desk-div"></div>
+                        <div class="col-5 image-item">
+                            <img src="images/about_2013.png" alt="about image">
+                        </div>
+
+                        <div class="col-5 image-item">
+                            <img src="images/about_2011.png" alt="about image">
+                        </div>
+
+                    </div>
+                </div>
+
+
+
+                <!--two photos in mobile version-->
+                <div class="container-fluid images-section" id="hide_images">
+                    <div class="row img-mobile">
+
+                        <div class="col-12 image-item">
+                            <img src="images/about_2013.png" alt="about image">
+                        </div>
+
+                        <div class="col-12 image-item">
+                            <img src="images/about_2011.png" alt="about image">
+                        </div>
+
+                    </div>
+                </div>
+                <!-- footer -->
+            <div class="container footer_about">
+                <div class="row">
+                    <div class="col-2"></div>
+                    <div class="col-2 footer-logo-1">
+                        <img src="./images/women.png" alt="">
```

Comment:
- [ Required ] Please consider adding a descritptive value for the  `alt` attribute for this image, this will make your web page accessible to persons using web accessibility tools like screen readers.